_shared/project/tox.ini

```diff
@@ -69,12 +70,13 @@ deps =
     format,checkformatting: black
     format,checkformatting: isort
     lint: toml
-    lint: pylint
+    lint: pylint>=3.0.0
     lint: pydocstyle
     lint: pycodestyle
     lint,tests: pytest-mock
-    lint,tests,coverage: coverage[toml]
     lint,tests,functests: pytest
+    tests: pytest-coverage
+    clean,coverage: coverage[toml]
```

Comment:
The new `clean` environment and the `coverage` environment both need the standard `coverage` command line tool (not `pytest-coverage`: that's only needed for running the tests)